fs/btrfs/zstd.c

```diff
@@ -421,6 +459,13 @@ static int zstd_decompress(struct list_head *ws, unsigned char *data_in,
 
 static void zstd_set_level(struct list_head *ws, unsigned int type)
 {
+	struct workspace *workspace = list_entry(ws, struct workspace, list);
+	unsigned level = (type & 0xF0) >> 4;
```

Comment:
nit: `const`.

Reply:
Right now level would be updated to ZSTD_BTRFS_MAX_LEVEL if it's greater than ZSTD_BTRFS_MAX_LEVEL. I assume we shouldn't make it const?

Reply:
Good point!